testcontainers/src/core/containers/async_container.rs

```diff
@@ -184,26 +185,50 @@ where
         self.docker_client.docker_hostname().await
     }
 
-    pub async fn exec(&self, cmd: ExecCommand) {
-        let ExecCommand {
+    pub async fn exec_with(&self, cmd: ExecWithCommand) -> LogStreamAsync<'_> {
```

Comment:
`LogStreamAsync` seems to be crate-level structure, we need to expose it, don't we?